extension/src/entrypoints/content.ts

```diff
@@ -0,0 +1,29 @@
+export default defineContentScript({
+  matches: ["https://meet.google.com/*"],
+  runAt: "document_start",
+  main() {
+    const s = document.createElement("script");
+    s.src = browser.runtime.getURL("/inject.js");
+    (document.head || document.documentElement).appendChild(s);
+    s.onload = () => s.remove();
+
+    window.addEventListener("message", async (event) => {
```

Comment:
nit: I think this might be most effectively read as its own named function, `handleMicMuted` for example